src/main/java/card/issuer/CardIssuers.java

```diff
@@ -0,0 +1,39 @@
+package main.java.card.issuer;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CardIssuers {
+    private static Map<String, CardIssuerParameters> cardIssuersMap =  new HashMap<String, CardIssuerParameters>();
+
+    public static boolean doesIssuerExist(String issuerName) {
+        if(cardIssuersMap.get(issuerName) == null)
+            return false;
+        else
+            return true;
+    }
+
+    public static CardIssuerParameters getIssuerParameters(String issuerName){
+        return cardIssuersMap.get(issuerName);
+    }
+
+    public static void prepareCardIssuersMap()
+    {
+        //EXAMPLE DATA(in future it may be created from a file)
+
+        ArrayList<Integer> digitNumber = new ArrayList<Integer>();
+        //MasterCard
+        digitNumber.add(16);
+        CardIssuerParameters masterCardParamters = new CardIssuerParameters("MasterCard", digitNumber);
```

Comment:
Use one-liner `List.of(16)` to initialize a List, that makes code more clear and short
```suggestion
        CardIssuerParameters masterCardParamters = new CardIssuerParameters("MasterCard", List.of(16));
```